components/phase-banner/assets/PhaseBanner.scss

```
@@ -5,6 +5,11 @@
   &__tag {
     @extend .govuk-phase-banner__content__tag;
   }
+  &__content {
+    p {
+      max-width: none;
+    }
+  }
```

```suggestion
  &__content {
    max-width: none;
  }
```

I think it can go straight onto `__content` can't it?